pkg/pmctl/cmd/root.go

```diff
@@ -0,0 +1,32 @@
+package cmd
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/spf13/cobra"
+)
+
+var globalParams struct {
+	endpoint string
+}
+
+// rootCmd represents the base command when called without any subcommands
+var rootCmd = &cobra.Command{
+	Use:   "pmctl",
+	Short: "control nodes, pods, and networks on placemat",
+	Long:  `pmctl is a command-line tool to control nodes, pods, and networks on placemat`,
```

Comment:
https://github.com/cybozu-go/well/wiki/Use-with-spf13-cobra#what-every-cobra-program-need-to-do